akka-sample-sharding-java/build.sbt

```diff
@@ -27,7 +27,7 @@ lazy val killrweather = project
   .in(file("killrweather"))
   .settings(commonSettings)
   .settings(
-    mainClass in (Compile, run) := Some("sample.killrweather.KillrWeather"),
+    run / mainClass := Some("sample.killrweather.KillrWeather"),
```

Comment:
Shouldn't this be `Compile / run / mainClass`?

Reply:
changed